test/searchable-words.js

```diff
@@ -0,0 +1,15 @@
+import test from 'ava';
+import {getSearchableWords as fn} from '../source/features/display-issue-suggestions';
+
+test('Searchable words filter', t => {
+	t.deepEqual(fn(' n o t h i n g	n e s s'), []);
+	t.deepEqual(fn('Hello, world!'), ['Hello', 'world']);
+	t.deepEqual(fn(' 400  	[error] is\\ nonsense !'), ['400', 'error', 'nonsense']);
+	t.deepEqual(fn('1col is broken'), ['1col', 'broken']);
+	t.deepEqual(fn('ava is 🔥🔥🔥'), ['ava']);
+	t.deepEqual(fn('1身 分'), ['1身', '分']);
+	t.deepEqual(fn('làm 7/2 nộp'), ['làm', 'nộp']);
+	t.deepEqual(fn('测试'), ['测试']);
+	t.deepEqual(fn('حمید'), ['حمید']);
```

Comment:
Stray brackets instead of parens here

Reply:
Oh… how did this test pass??

Reply:
hahahahahahahahah oh Allah.

Arabic is a RTL language and GitHub's syntax highlighter can't handle/contain the RTL change. Copy and paste the code into your editor and you'll see the correct order (works with Sublime)